Source/src/modules/ModuleSceneManager.cpp

```diff
@@ -54,6 +53,19 @@ void Hachiko::ModuleSceneManager::AttemptScenePause()
 
 void Hachiko::ModuleSceneManager::AttemptScenePlay()
 {
+    ComponentCamera* scene_camera = main_scene->GetMainCamera();
+
+    if (scene_camera == nullptr)
+    {
+        HE_LOG("Current scene does not have a CameraComponent inside."
+               " Therefore, cannot enter Play Mode.");
```

Comment:
why are the lines splitted like this? was the code formatted? I would say it's weird the formatter is configured that like in the repo

Reply:
I've been using 80 chars limit myself, so didn't think about it that much, if you think this is a problem, I can make it one single line. 👍

Reply:
I'm going to drop some hardcore advice here that a former lead gave me a while back, "we are not developing in the past, modern monitors support greater resolutions and are widescreen, 80 chars limit is for when we developed in text mode, on 4:3 monitors", we can easly support 150, 200 chars per line and still be able to see the whole code

Reply:
Yeah you are probably right, it's just me feeling more comfortable with it after getting used to default settings for dart/flutter.